src/utils/utils.js

```diff
@@ -63,7 +63,11 @@ Utils.buildCCSSFn = function(js, attr) {
 };
 
 Utils.buildAndExecuteFn =  function (js) {
-  return Utils.buildCCSSFn(js, ['$zoom'])(10);
+  return Utils.buildCCSSFn(
+    R.map(
```

Comment:
What I am doing here is replace every signal of the `mapnik::geometry_type` variable, since tangram doesn't have some features dynamically... We can't know that value at loading time, so we make as if it never exists replacing it with a true value.

Now, we can run that function without any information to have our static value under dynamic conditions.